src/export/script.js

```diff
@@ -0,0 +1,42 @@
+document.addEventListener('DOMContentLoaded', function() {
+  const allLines = document.querySelectorAll('body > div');
+  const pageBreakMarkers = document.querySelectorAll('.container');
+  let currentPageIndex = 0;
+
+
+  function showCurrentPage() {
+    allLines.forEach((line) => {
+      line.classList.add('hidden');
+    });
+
+    allLines[currentPageIndex].classList.remove('hidden');
+  }
+
+
+  function scaler() {
+    var w = document.documentElement.clientWidth;
+    let scaledAmount= w/829;
```

Comment:
What is this magical number?

Reply:
it's the original width of the presentation, I left it in there for testing
I forgot to actually implement it :shrug: